template-parts/home-info.html

```diff
@@ -0,0 +1,46 @@
+<h2>Info Just for You</h2>
+<div class="wp-block-columns has-2-columns info-block">
```

Comment:
I was initially thinking that we'd do all of this as widgets, but I think using blocks might actually be better 👍 

That'd make it much easier for anyone to edit the content without messing up the markup, which could be a problem w/ the HTML widget.

I'm not sure exactly how we'll get editor styles working, but there should be a way 🤔 

* https://developer.wordpress.org/block-editor/developers/themes/theme-support/#editor-styles

Maybe we can propose adding an extra field to [the Remote CSS screen](https://2019.seattle.wordcamp.org/wp-admin/themes.php?page=remote-css) for a separate CSS file, and if that's filled in, then it'd be enqueued in the editor?

Related:

* https://meta.trac.wordpress.org/ticket/4392
* https://meta.trac.wordpress.org/ticket/4389

I don't think that having editor styles is a blocker, though, we can definitely move forward and work on it later. I do worry, though, that if we don't eventually add them then people might make destructive edits because they won't be able to see the impact that the changes will make until after they've saved and switched to the front end (unless they use the preview feature, but they probably wouldn't realize that they need to).